x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings_by_resource/use_findings_by_resource.ts

```diff
@@ -14,7 +14,7 @@ import { showErrorToast } from '../latest_findings/use_latest_findings';
 import type { FindingsBaseEsQuery, FindingsQueryResult } from '../types';
 
 // a large number to probably get all the buckets
-const MAX_BUCKETS = 60 * 1000;
+const MAX_BUCKETS = 1000 * 1000;
```

Comment:
FYI this is [limited](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-settings.html#search-settings-max-buckets) by `search.max_buckets` which has a default of `65,536`

Reply:
thanks, I will fix it